nats_test.go

```diff
@@ -1319,3 +1321,112 @@ func TestNkeyAuth(t *testing.T) {
 		t.Fatalf("Error on Flush: %v", err)
 	}
 }
+
+func createTmpFile(t *testing.T, content []byte) string {
+	t.Helper()
+	conf, err := ioutil.TempFile("", "")
+	if err != nil {
+		t.Fatalf("Error creating conf file: %v", err)
+	}
+	fName := conf.Name()
+	conf.Close()
+	if err := ioutil.WriteFile(fName, content, 0666); err != nil {
+		os.Remove(fName)
+		t.Fatalf("Error writing conf file: %v", err)
+	}
+	return fName
+}
+
+func TestNKeyOptionFromSeed(t *testing.T) {
+	if _, err := NkeyOptionFromSeed("file_that_does_not_exist"); err == nil {
+		t.Fatal("Expected error got none")
+	}
+
+	seedFile := createTmpFile(t, []byte(`
+# No seed
```

Comment:
Since parser can deal with spaces etc maybe just do inline since bytes and not have indentation.